*** /usr/local/xwiki/web.xml.orig	2023-06-29 21:01:52.043824600 +0000
--- web.xml	2023-06-29 21:03:52.065361881 +0000
***************
*** 50,55 ****
--- 50,68 ----
      <filter-class>org.xwiki.resource.servlet.RoutingFilter</filter-class>
    </filter>
  
+   <!-- URL rewriting filter (for short URLs) -->
+   <filter>
+     <filter-name>UrlRewriteFilter</filter-name>
+     <filter-class>org.tuckey.web.filters.urlrewrite.UrlRewriteFilter</filter-class>
+     <logLevel>sysout:DEBUG</logLevel>
+    </filter>
+    <filter-mapping>
+      <filter-name>UrlRewriteFilter</filter-name>
+      <url-pattern>/*</url-pattern>
+      <dispatcher>REQUEST</dispatcher>
+      <dispatcher>FORWARD</dispatcher>
+    </filter-mapping>
+ 
    <!-- Filter that sets a custom encoding to all requests, since usually clients don't specify
         the encoding used for submitting the request, so by default containers fall back to the
         encoding globally configured in their settings. This allows XWiki to use a custom encoding,
